contentcuration/contentcuration/models.py

```diff
@@ -412,11 +412,22 @@ def get_user_active_trees(self):
     def get_user_active_files(self):
         cte = With(self.get_user_active_trees().distinct())
 
-        return (
-            cte.join(self.files.get_queryset(), contentnode__tree_id=cte.col.tree_id)
-            .with_cte(cte)
-            .values("checksum")
-            .distinct()
+        files_qs = cte.join(
+            self.files.get_queryset(), contentnode__tree_id=cte.col.tree_id
+        ).with_cte(cte)
+
+        files_qs = self._filter_storage_billable_files(files_qs)
+
+        return files_qs.values("checksum").distinct()
+
+    def _filter_storage_billable_files(self, queryset):
+        """
+        Perseus exports would not be included in storage calculations.
+        """
+        if queryset is None:
+            return queryset
+        return queryset.exclude(file_format_id__isnull=True).exclude(
+            file_format_id=file_formats.PERSEUS
```

Comment:
Not an immediate concern, but just a heads that when QTI assessments are more broadly available, and we are generating QTI ZIP files, then we may need to filter these too (and it would need to be on the format preset, rather than the file format id, because the format id would be 'zip'!)